keps/sig-node/693-topology-manager/README.md

```diff
@@ -929,3 +930,11 @@ allocation and thread scheduling, but does not address device locality.
 
 Multi-NUMA hardware is needed for testing of this feature. Recently, support for multi-NUMA
 harware was [added](https://github.com/kubernetes/test-infra/pull/28369) in Kubernetes test infrastructure.
+
+## Windows considerations
+
+Topology manager is already enabled on Windows in order to support the device manager.  The same configuration options
+and PRR applies to Windows. The CPU manager and Memory Manager can independently be enabled to support advance configuration
```

Comment:
what is PRR in this case?

Reply:
I meant the same Production Readiness Review answers, in the sense that these features can be set to `None` to disable if required to roll back do to errors.   I updated to clarify